vividus-plugin-applitools/src/main/java/org/vividus/visual/eyes/VisualTestingSteps.java

```diff
@@ -0,0 +1,189 @@
+     *        <td>https://eyesapi.applitools.com</td>
+     *        <td>AUT</td>
+     *        <td>UAT</td>
+     *        <td>PREDEV</td>
+     *        <td>someName</td>
+     *        <td>ESTABLISH</td>
+     *      </tr>
+     *    </tbody>
+     *  </table>
+     *  <br> Mind that: batchName, baselineName, action are mandatory parameters;
+     * @param applitoolsVisualChecks custom eyes configuration
+     */
+    @When("I run eyes test using:$eyesConfiguration")
+    public void performCheck(List<ApplitoolsVisualCheck> applitoolsVisualChecks)
+    {
+        applitoolsVisualChecks.forEach(this::runApplitoolsTest);
+    }
+
+    /**
+     * Perform visual check on the Applitools service and provides possibility to use custom configurations for
+     * the checks and screenshot strategy;<br>
+     * <table border="1">
+     *    <caption>Example of eyes configuration:</caption>
+     *    <thead>
+     *      <tr>
+     *        <th>readApiKey</th>
+     *        <th>hostApp</th>
+     *        <th>hostOS</th>
+     *        <th>viewportSize</th>
+     *        <th>matchLevel</th>
+     *        <th>serverUri</th>
+     *        <th>appName</th>
+     *        <th>batchName</th>
+     *        <th>baselineEnvName</th>
+     *        <th>baselineName</th>
+     *        <th>action</th>
+     *      </tr>
+     *    </thead>
+     *    <tbody>
+     *      <tr>
+     *        <td>key</td>
+     *        <td>Chrome</td>
+     *        <td>Windows 95</td>
+     *        <td>1x1</td>
+     *        <td>EXACT</td>
+     *        <td>https://eyesapi.applitools.com</td>
+     *        <td>AUT</td>
+     *        <td>UAT</td>
+     *        <td>PREDEV</td>
+     *        <td>someName</td>
+     *        <td>ESTABLISH</td>
+     *      </tr>
+     *    </tbody>
+     *  </table>
+     *  <br> Mind that: batchName, baselineName, action are mandatory parameters;<br>
+     * <table border="1">
+     *    <caption>Example of screenshot configuration:</caption>
+     *    <thead>
+     *      <tr>
+     *        <th>webHeaderToCut</th>
+     *        <th>webFooterToCut</th>
+     *        <th>nativeHeaderToCut</th>
+     *        <th>nativeFooterToCut</th>
+     *        <th>scrollableElement</th>
+     *        <th>coordsProvider</th>
+     *        <th>scrollTimeout</th>
+     *        <th>screenshotShootingStrategy</th>
+     *      </tr>
+     *    </thead>
+     *    <tbody>
+     *      <tr>
+     *        <td>100</td>
+     *        <td>10</td>
+     *        <td>20</td>
+     *        <td>20</td>
+     *        <td>By.id(scrollable)</td>
+     *        <td>WEB_DRIVER</td>
+     *        <td>500</td>
+     *        <td>SIMPLE</td>
+     *      </tr>
+     *    </tbody>
+     *  </table>
+     * @param applitoolsVisualChecks custom eyes configuration
+     * @param screenshotConfiguration custom shooting strategy configuration
+     */
+    @When(value = "I run eyes test using:$eyesConfiguration and screenshot config:$screenshotConfiguration",
```

Comment:
actually testers do not know what means `eyes` here, does it make sense to rename it to applitools?